af/shovel/oonipl/pg.py

```diff
@@ -214,7 +214,7 @@ def pg_binquote(s):
 def pg_uniquote(s):
     if isinstance(s, str):
         s = unicode(s, "utf-8")
-    assert isinstance(s, unicode)
+    assert isinstance(s, unicode), "{} not unicode".format(s)
```

Comment:
`"%r is not unicode" % s`   to show the repr of s

Reply:
Ok